client/containers/DashboardEdges/DashboardEdges.tsx

```diff
@@ -133,7 +133,7 @@ const DashboardEdges = (props: Props) => {
 					<NonIdealState
 						icon="layout-auto"
 						title="No connections from other Pubs"
-						description="When other authors add connections to this Pub, they will be listed here."
+						description="When other contributors add connections from their Pubs to this Pub, they will be listed here."
```

Comment:
Not a blocker, but this one strikes me as slightly confusing. Contributors can't technically add connections from their Pubs. Only Pub Admins can. So perhaps we should say "When other users add connections from their Pubs..."

Reply:
I was tempted to go with `members` here, as you can only use predictive search to connect to Pubs within your own Community. But connections are still made between Pubs in separate Communities when you enter the Pub URL or DOI, so the Community-restricted term doesn't quite encompass all use cases. (I'm just writing this out in case there's an error in my logic/understanding.) I did invert the syntax of the sentence to remove the conditional structure and make it a touch more descriptive. I also made it "users" instead of "other users," because a user could create connections between two or more Pubs that they administer. And because having permissions to make connections doesn't necessarily assume ownership, I removed the possessive, "their." So, we end up with: `Corresponding connections will be created here when users connect to this Pub from another.`